openmetadata-ui/src/main/resources/ui/src/utils/EntityUtilClassBase.test.ts

```diff
@@ -109,7 +110,11 @@ describe('EntityUtilClassBase', () => {
     const fqn = 'testingGlossary';
     entityUtil.getEntityLink(EntityType.GLOSSARY, fqn);
 
-    expect(getGlossaryPath).toHaveBeenCalledWith(fqn);
+    expect(getGlossaryTermDetailsPath).toHaveBeenCalledWith(
+      fqn,
+      undefined,
+      undefined
+    );
   });
```

Comment:
Aren't we need to test the return value of it ? With the tab, subTab and only with the fqn.

Reply:
Return values will be verified with test of that functions, Here we just need to verify the correct function call  with params based on entityType